components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -801,40 +814,37 @@ - (void)cacheLayoutCalculations {
 - (void)cacheLayoutCalculationsWithAddedContentHeight:(CGFloat)addedContentHeight {
   CGFloat contentHeaderHeight = self.contentHeaderHeight;
   CGFloat containerHeight = self.presentingViewBounds.size.height;
-  CGFloat contentHeight = self.contentViewController.preferredContentSize.height +
-                          [self bottomSafeAreaInsetsToAdjustContainerHeight];
+  CGFloat contentHeight = [self contentVCPreferredHeight];
   if ([self shouldPresentFullScreen]) {
     contentHeight = MAX(contentHeight, containerHeight - self.topHeaderHeight);
   }
   _contentVCPreferredContentSizeHeightCached = contentHeight;
 
   contentHeight += addedContentHeight;
   _addedContentHeight = addedContentHeight;
+  _maximumInitialDrawerHeightCalculated = [self calculateMaximumInitialDrawerHeight];
 
   CGFloat totalHeight = contentHeight + contentHeaderHeight;
-  BOOL contentScrollsToReveal = totalHeight >= self.maximumInitialDrawerHeight;
-
-  if (_contentHeaderTopInset == NSNotFound) {
-    // The content header top inset is only set once.
```

Comment:
What is the context for only calculating this once? I haven't noticed any side effects of recalculating every time the sheet needs an updated cache, but I assume it was this way for a reason I'm not aware of.

Reply:
I don't believe there is any reason to only do it once aside from performance reasons. If this is called multiple times it may be slower? But I don't see this calculation being meaty at all.